light-jobs-test/src/main/java/com/hazelcast/jet/tests/lightjobs/LightJobsTest.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.lightjobs;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.pipeline.JournalInitialPosition;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.Sources;
+import com.hazelcast.jet.tests.common.AbstractSoakTest;
+import com.hazelcast.map.IMap;
+
+import static com.hazelcast.jet.tests.common.Util.sleepMillis;
+
+public class LightJobsTest extends AbstractSoakTest {
+
+    private static final String SOURCE_BATCH_MAP_NAME = "lightJobBatchInputMap";
+    private static final String SOURCE_JOURNAL_MAP_NAME = "lightJobJournalInputMap";
+    private static final String SOURCE_INCORRECT_MAP_NAME = "lightJobIncorrectMap";
+    private static final String OUTPUT_BATCH_MAP_PREFIX = "lightJobBatchOutputSinkMap";
+    private static final String OUTPUT_STREAM_MAP_PREFIX = "lightJobJournalOutputSinkMap";
+    private static final String OUTPUT_INCORRECT_MAP_NAME = "lightJobIncorrectSinkMap";
+
+    private static final String EXPECTED_FAIL_MESSAGE = "LightJobsTest - incorrectPipeline - expected exception";
+
+    private static final int DEFAULT_ITEM_COUNT = 100;
```

Comment:
we can increase default item count to 300-400. our partition count is 64 afaik with 100 items there may be some partitions without any items.